examples/plot_anomaly_comparison.py

```diff
@@ -0,0 +1,120 @@
+"""
+========================================================================
+Comparing different anomaly/novelty detection algorithms on toy datasets
+========================================================================
+
+This example shows characteristics of different
+anomaly/novelty detection algorithms on 2D datasets. Datasets contain
+one or two modes (regions of high density) to illustrate the ability
+of algorithms to cope with multimodal data.
+
+While these examples give some intuition about the
+algorithms, this intuition might not apply to very high
+dimensional data.
+"""
+print(__doc__)  # noqa
+
+import time
+
+import numpy as np
+from scipy import stats
+import matplotlib
+import matplotlib.pyplot as plt
+
+from sklearn import svm
+from sklearn.datasets import make_moons
+from sklearn.covariance import EllipticEnvelope
+from sklearn.ensemble import IsolationForest
+from sklearn.neighbors import LocalOutlierFactor
+
+matplotlib.rcParams['contour.negative_linestyle'] = 'solid'
+
+# Example settings
+n_samples = 300
+outliers_fraction = 0.15
+clusters_separation = (0, 1, 2)
+n_outliers = int(outliers_fraction * n_samples)
+n_inliers = n_samples - n_outliers
+
+
+def make_dataset(n_inliers, offset, stds=(0.5, 0.5), random_state=0):
```

Comment:
This is a special case of make_blobs, right? Well, here we have explicit offsets, but you could just pick a nice-looking random state for make_blobs?